app/controlplane/pkg/data/ent/migrate/schema.go

```diff
@@ -631,6 +631,11 @@ var (
 			},
 		},
 		Indexes: []*schema.Index{
+			{
```

Comment:
why the query doesn't take into account `workflowrun_workflow_id_created_at` instead?

Reply:
I guess it needs both fields to be present in the query. Here it's just using `created_at` (in the ORDER_BY)